### ISSUE TITLE
Illegal division by zero at data/script/ROUGE-1.5.5.pl line 2450

### ISSUE THREAD

**Issue by SherryW0424** [lancopku/superAE#11]
您好，我使用的是lcsts2.0数据集，根据前面issue里面提到的将数据集里的文本和摘要分别放入src以及tgt文件中，但在运行rouge的时候报错，查阅其他资料好像是因为采用中文字符的原因？ 但本篇文章就是训练的中文摘要啊？ 
      此外，candidate文件夹里每个文件都是<unk>，不知道这是正常的，还是因为我哪个步骤做错了？

log如下：
/mnt/extend_sdb/workspace/superAE/superAE/models/attention.py:32: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  weights = self.softmax(weights)   # batch * time
/mnt/extend_sdb/workspace/superAE/superAE/models/loss.py:190: UserWarning: self and other not broadcastable, but have the same number of elements.  Falling back to deprecated pointwise behavior.
  num_correct = pred.data.eq(targets.data).masked_select(targets.ne(dict.PAD).data).sum()
/mnt/extend_sdb/workspace/superAE/superAE/models/loss.py:190: UserWarning: self and mask not broadcastable, but have the same number of elements.  Falling back to deprecated pointwise behavior.
  num_correct = pred.data.eq(targets.data).masked_select(targets.ne(dict.PAD).data).sum()
 [======================================== 10000/10000 ================================>]  Step: 3s712ms | Tot: 6h2m                                           
epoch:   1, ppl:  1.006, time: 21772.570, updates:    10000, accuracy: 95.07
evaluating after 10000 updates...
/mnt/extend_sdb/workspace/superAE/superAE/models/seq2seq.py:169: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  output = unbottle(self.log_softmax(output))
Illegal division by zero at data/script/ROUGE-1.5.5.pl line 2450........................]  Step: 410ms | Tot: 0ms                                              
F_measure: [0.0, 0.0, 0.0] Recall: [0.0, 0.0, 0.0] Precision: [0.0, 0.0, 0.0]

运行环境：
python 3.5  pytorch0.3.1 cuda9.0

**Comment by angeluau** [comment]
现在调试好了没

**Comment by SherryW0424** [comment]
没有

**Comment by jufengada** [comment]
您好，请问你现在调试好了没有？

**Comment by ishurironaldinho** [comment]
您好，请问将文本和摘要分别放入src和tgt这一步该怎么完成???

**Comment by jufengada** [comment]
我也是这样，请问你现在这个问题解决了吗？

**Comment by jufengada** [comment]
epoch:   1, ppl:  1.006, time: 20976.414, updates:    10000, accuracy: 95.00
F_measure: [0.0, 0.0, 0.0] Recall: [0.0, 0.0, 0.0] Precision: [0.0, 0.0, 0.0]

**Comment by Edith-GHX** [comment]
你好，请问现在解决了吗
